Adds data collection notice to CLI page (#4296)
FusionAuth/fusionauth-site: astro/src/content/docs/customize/cli.mdx

  - `fusionauth theme:upload` - Upload a theme to a FusionAuth server.
  - `fusionauth theme:watch` - Watch a theme directory and upload changes to a FusionAuth server.

## Usage data collection

By default, the FusionAuth CLI collects data about your usage. We use this to judge effectiveness of the CLI, check performance and reliability, and help drive new features and improvements.

> If you'd prefer not to send usage data, run the following command:

Reviewer: ````suggestion
If you'd prefer not to send usage data, run the following command:

```console
npx fusionauth telemetry:disable
```

If you change your mind and decide to share usage data with us, run this command:

```console
npx fusionauth telemetry:enable
```

````

  - `fusionauth theme:upload` - Upload a theme to a FusionAuth server.
  - `fusionauth theme:watch` - Watch a theme directory and upload changes to a FusionAuth server.

## Usage data collection

> By default, the FusionAuth CLI collects data on usage. We use this to judge effectiveness of the CLI, check performance and reliability, and help drive new features and improvements.

Reviewer: ```suggestion
By default, the FusionAuth CLI collects data about your usage. We use this to judge effectiveness of the CLI, check performance and reliability, and help drive new features and improvements.
```

If you'd prefer not to send usage data, you can disable telemetry by editing the `telemetry` with the following command:

```sh
# opt out of sharing usage data
npx fusionauth telemetry:disable

# opt back into sharing usage data
npx fusionauth telemetry:enable

> ```

Reviewer: ````suggestion
If you'd prefer not to send usage data, run the following command:

```console
npx fusionauth telemetry:disable
```

If you change your mind and decide to share usage data with us, run this command:

```console
npx fusionauth telemetry:enable
```
````
suggestion: switch these to interactive console snippets, break them up into two different code blocks so users can use our copy buttons

> 
> If you'd prefer not to send usage data, you can disable telemetry by editing the `telemetry` with the following command:
> 
> ```sh
> # opt out of sharing usage data
> npx fusionauth telemetry:disable
> 
> # opt back into sharing usage data
> npx fusionauth telemetry:enable
> ```
> 
> 
> 

Reviewer: ```suggestion
```
remove some extra newlines
